src/app/game/board/deck/deck.component.js

```diff
@@ -11,26 +11,39 @@ class Deck extends Component {
             drawPile: this.props.drawPile
         };
 
-        this.moveCard = this.moveCard.bind(this);
+    /*    this.moveCard = this.moveCard.bind(this);*/
```

Comment:
please don't leave any commented out lines in code you want to push

Reply:
ok, its just that im used to not wanting  to delete code unless you authorise it and i want you to see what code was there before.  but thats all the meaning of using GIT. :-) 
Eventually i will implement it .